params/precompile_config_test.go

```diff
@@ -64,7 +65,7 @@ func TestVerifyWithChainConfig(t *testing.T) {
 }
 
 func TestVerifyWithChainConfigAtNilTimestamp(t *testing.T) {
-	admins := []common.Address{{1}}
+	admins := []common.Address{allowlist.TestAdminAddr}
```

Comment:
I think it would be nicer if params did not import allowlist

Reply:
yep good idea. actually `allowlist` is already in the import path for params tests as we import other precompile packages. but i agree, we should not import more precompile packages here.